src/riot-rs-runqueue/src/runqueue.rs

```diff
@@ -214,6 +258,17 @@ mod clist {
         }
     }
 
+    /// Helper function that is needed because hax doesn't support `Iterator::position` yet.
+    fn position<const N: usize>(slice: &[u8; N], search_item: u8) -> Option<usize> {
+        let mut index = None;
+        for i in 0..N {
+            if slice[i] == search_item {
+                index = Some(i);
```

Comment:
```suggestion
                index = Some(i);
                break;
```

Reply:
(... or just `return Some(i)`, drop the index variable, and have a `None` in line 269. preference.)

Reply:
Early returns from loops are unfortunately not supported by hax yet, that's why this work-around is needed :/

Reply:
Done in 3fcff18.